_data/docs.yml

```
@@ -60,14 +60,25 @@ apis:
   cuspatial:
     name: cuspatial
     path: cuspatial
-    desc: 'cuSpatial is a GPU accelerated C++/Python library for accelerating GIS workflows including point-in-polygon, spatial join, coordinate systems, shape primitives, distances, and trajectory analysis.'
+    desc: 'cuSpatial is a GPU-accelerated vector GIS library including binary predicates (DE-9IM), point-in-polygon, spatial join, distances, and trajectory analysis.'
     ghlink: https://github.com/rapidsai/cuspatial
     cllink: https://github.com/rapidsai/cuspatial/blob/main/CHANGELOG.md
     versions:
       # enable or disable links; 0 = disabled, 1 = enabled
       legacy: 1
       stable: 1
       nightly: 1
+  cuproj:
+    name: cuProj
+    path: cuProj
+    desc: 'cuProj is a GPU-accelerated geographic and geodetic coordinate transformation library which supports projecting coordinates between coordinate reference systems (CRSes), compatible with PyProj.'
+    ghlink: https://github.com/rapidsai/cuspatial/python/cuproj
```

This link is a 404. Once the release PR is merged, this will be the stable link you want:
```suggestion
    ghlink: https://github.com/rapidsai/cuspatial/tree/main/python/cuproj
```